portal-impl/src/com/liferay/portlet/asset/service/impl/AssetCategoryLocalServiceImpl.java

```diff
@@ -415,6 +430,52 @@ public List<AssetCategory> getEntryCategories(long entryId) {
 		return Collections.emptyList();
 	}
 
+	@Override
+	public AssetCategory getOrAddIncompleteCategory(
```

Comment:
I don't know if there are some already consolidate names but this should not be `getOrAddIncompleteAssetCategory` ? I can see that the fetch below (line 438) has this name

Reply:
Hi,
 the names on all the method on the services are without the asset.
 So in my opinion, If I put getOrAddIncompeAsset*, Brian will ask for a change

Reply:
sure, I wrote a note on this just because for our services usually the name of the method should reflect 1..1 the name of the model.